web/client/src/app.rs

```diff
@@ -39,7 +40,7 @@ pub fn MainApp(_hooks: &mut Hooks, server_url: String) -> Element {
             let world = &mut game_state.world;
 
             // TODO: client side wasm on the web
```

Comment:
Not really a TODO any more, is it? 😜

Reply:
Yes, it feels so good to finally have this big question mark shaped roadblock out of the way